doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_custom_pcb.rst

```diff
@@ -110,6 +114,8 @@ The presence of a ``bicr`` node in the application devicetree will automatically
 Flashing the BICR binary
 ========================
 
+##TODO
+
 After the |NCS| build system generates the BICR binary, you must flash this binary manually.
 The content of BICR should be loaded to the SoC only once and should not be erased nor modified unless the PCB layout changes.
 To manually flash the generated :file:`bicr.hex` file to the SoC, use nRF Util as follows::
```

Comment:
bicr.hex can be programmed like this
`nrfutil device program --options chip_erase_mode=ERASE_NONE --firmware bicr.hex --core Application --serial-number <DK_SERIAL_NUMBER>`

but I'm not sure if west flashwill not do it also.

Reply:
Is the --core parameter supposed to be ``application`` instead of ``secure``?

Reply:
Yes secure can be used only for secdom and when LCS is empty. In ROT You will get error when trying to use --core secure

Reply:
> but I'm not sure if west flashwill not do it also.
I dont think west flash should program anything that is likely to brick your device. I think this should be kept nrfutil only.